corehq/apps/users/tasks.py

```diff
@@ -294,7 +294,7 @@ def remove_unused_custom_fields_from_users_task(domain):
 @task
 def bulk_deactivate_users(domain):
     from corehq.apps.users.dbaccessors import get_all_commcare_users_by_domain
-    users = get_all_commcare_users_by_domain(domain)
+    users = get_all_commcare_users_by_domain(domain.name)
```

Comment:
I'd just pass in the domain name if you don't need the full object.  That'll minimize the state that needs to be passed between these things.  Also, we have a loose convention that `domain` refers to the string domain name and `domain_obj` (or `project` in older code) refers to the domain object.  That's definitely not always held to, but I'd recommend using it for new code.